### ISSUE TITLE
Firebase (again !)

### ISSUE THREAD

**Issue by marcoherbst** [evercam/evercam-android#204]
I was talking to Google Analytics team. 
We currently have 2 Accounts (Evercam , Evercam Mobile)
In Evercam we have 2 views (All Website , User by ID)
In Evercam Mobile we have 4 properties (Admob, All, iOS, Android)

They suggest it would be better to have just 1 Account and to add a 3rd Property (Mobile) using the Firebase SDK. 

Sounds sensible to me, in which case, the task is to add that for both iOS & Android including (important!) the goals that are nicely setup by Liuting.

**Comment by marcoherbst** [comment]
Further info: When we're doing this, take into account the "User by ID" config. Here was what Google Support said:

- This view was initially set up by enabling the "Show User ID reports" section of the settings (you'll see this in the "Admin" tab under the View section in "View settings". 
- The webmaster or web developer would have then edited the tracking code to send the ID through to Google Analytics - here is the guide to how that is done. 
- The user explorer section is in Google Analytics by default so will be present in both the "All web site data" and "User by ID" views. It displays the ID that Analytics will assign to each device that enters the web site. The difference between the two is that User ID can access cross-device reports. These are not available in any other views. 
- Because the web developer set up User IDs for you, that is why you can see the username they use on the site within Analytics User by ID view. 
- In short, user ID will allow you to assign one user ID to a person who has entered the site on both mobile and desktop, for example. This makes data far more accurate and gives you the ability to assign an identity to the user based on their account information with you on the site itself.
